### PR TITLE
fix: terms and editorials for better readability

### PR DESCRIPTION
**Pull request by peppelinux**
According to @paulbastian review

### CODE REVIEW

**Review by fmarino-ipzs — APPROVED** [review]

**Review by marinaado — COMMENTED** [review]
I am ok with the change, but we need to be consistent with how we defined the terminology and use the capitalised terms then when we refer to terms in the terminology:  
- option 1: we keep Issuer and Verifier in the terminology and refer it as credential Issuer and credential Verifier in the text
- option 2: we change the terminology yo Credential Issuer and Credential Verifier, and use that throughout the text.

**Review by paulbastian — APPROVED** [review]

**Comment by peppelinux** [comment]
@marinaado we don't capitalize terms until we don't have defined them in the terminology section (this is what I have learned in ietf/openid specs)

**Comment by marinaado** [comment]
> @marinaado we don't capitalize terms until we don't have defined them in the terminology section (this is what I have learned in ietf/openid specs)
Indeed, and those are part of the terminology section so this is why i suggested to capitalise them.

File: draft-demarco-oauth-status-assertions.md [hunk 1]
```diff
@@ -110,10 +110,10 @@ informative:
 Status Assertion is a signed object that demonstrates the validity status of a
 digital credential.
 These assertions are periodically provided
-to holders, who can present these to Verifier along
+to holders, who can present these to credential verifier along
```
Comment (peppelinux) [inline draft-demarco-oauth-status-assertions.md:113]:
```suggestion
to Holders, who can present these to Credential Verifier along
```

File: draft-demarco-oauth-status-assertions.md [hunk 2]
```diff
@@ -110,10 +110,10 @@ informative:
...
 with the corresponding digital credentials.
 The approach outlined in this document
-makes the Verifier able to check the status,
+makes the credential verifier able to check the status,
```
Comment (peppelinux) [inline draft-demarco-oauth-status-assertions.md:116]:
```suggestion
makes the Credential Verifier able to check the status,
```

File: draft-demarco-oauth-status-assertions.md [hunk 3]
```diff
@@ -124,46 +124,46 @@ without requiring to query any third-party entities.
 Status Assertions show the status of digital
 credentials, whether in JSON Web Tokens (JWT) or CBOR Web Tokens (CWT)
 format. Status Assertions function
-similarly to OCSP Stapling ([RFC6066]), allowing clients to present to the
-relying parties
-time-stamped assertions provided by the issuer.
+similarly to OCSP Stapling ([RFC6066]), allowing credential holders
```
Comment (peppelinux) [inline draft-demarco-oauth-status-assertions.md:127]:
```suggestion
similarly to OCSP Stapling ([RFC6066]), allowing Holders
```

File: draft-demarco-oauth-status-assertions.md [hunk 4]
```diff
@@ -124,46 +124,46 @@ without requiring to query any third-party entities.
...
+to present to the relying parties
+time-stamped assertions provided by the credential issuer.
```
Comment (peppelinux) [inline draft-demarco-oauth-status-assertions.md:129]:
```suggestion
time-stamped assertions provided by the Credential Issuer.
```

File: draft-demarco-oauth-status-assertions.md [hunk 5]
```diff
@@ -124,46 +124,46 @@ without requiring to query any third-party entities.
...
 The approach outlined in this specification enables the
 verification of credentials against revocation without
 direct queries to third-party systems,
 enhancing privacy, reducing latency, and
 faciliting offline verification.
 
-The figure below illustrates the process by which a client,
+The figure below illustrates the process by which a credential holder,
```
Comment (peppelinux) [inline draft-demarco-oauth-status-assertions.md:136]:
```suggestion
The figure below illustrates the process by which a Holder,
```

File: draft-demarco-oauth-status-assertions.md [hunk 6]
```diff
@@ -124,46 +124,46 @@ without requiring to query any third-party entities.
...
 such as a wallet instance,
 requests and obtains a Status Assertion from the issuer.
```
Comment (peppelinux) [inline draft-demarco-oauth-status-assertions.md:138]:
```suggestion
requests and obtains a Status Assertion from the Credential Issuer.
```

File: draft-demarco-oauth-status-assertions.md [hunk 7]
```diff
@@ -124,46 +124,46 @@ without requiring to query any third-party entities.
...
-|     Client     |                              |      Issuer      |
+|     Holder     |                              |      Issuer      |
 |                | Status Assertions            |                  |
 |                |<-----------------------------|                  |
 +----------------+                              +------------------+
 ~~~
 **Figure 1**: Status Assertion Issuance Flow.
 
-The figure below illustrates the process by which a client
+The figure below illustrates the process by which a credential holder
```
Comment (peppelinux) [inline draft-demarco-oauth-status-assertions.md:151]:
```suggestion
The figure below illustrates the process by which a Holder
```

File: draft-demarco-oauth-status-assertions.md [hunk 8]
```diff
@@ -124,46 +124,46 @@ without requiring to query any third-party entities.
...
 |                |---------------------------->|                  |
 +----------------+                             +------------------+
 ~~~
 **Figure 2**: Status Assertion Presentation Flow.
 
-In summary, the Issuer provides the client with a
-Status Assertion, which is linked to a Digital Credential. This enables
-the client to present both the digital credential and its
-Status Assertion to a Verifier as proof of the digital credential's
+In summary, the credential issuer provides the holder with a
```
Comment (peppelinux) [inline draft-demarco-oauth-status-assertions.md:163]:
```suggestion
In summary, the Credential Issuer provides the Holder with a
```

File: draft-demarco-oauth-status-assertions.md [hunk 9]
```diff
@@ -124,46 +124,46 @@ without requiring to query any third-party entities.
...
+Status Assertion, which is linked to a digital credential. This enables
```
Comment (peppelinux) [inline draft-demarco-oauth-status-assertions.md:164]:
```suggestion
Status Assertion, which is linked to a Digital Credential. This enables
```

File: draft-demarco-oauth-status-assertions.md [hunk 10]
```diff
@@ -124,46 +124,46 @@ without requiring to query any third-party entities.
...
+the holder to present both the digital credential and its
+Status Assertion to a credential verifier as proof of the digital credential's
```
Comment (peppelinux) [inline draft-demarco-oauth-status-assertions.md:166]:
```suggestion
the Holder to present both the Digital Credential and its
Status Assertion to a Credential Verifier as proof of the Digital Credential's
```

File: draft-demarco-oauth-status-assertions.md (hunk 1 above)
Comment (OR13) [reply draft-demarco-oauth-status-assertions.md:113]: I prefer "Verifier/Relying Party"

File: draft-demarco-oauth-status-assertions.md (hunk 4 above)
Comment (OR13) [reply draft-demarco-oauth-status-assertions.md:129]: I prefer "Issuer"

**Review by OR13 — APPROVED** [review]

File: draft-demarco-oauth-status-assertions.md [hunk 11]
```diff
@@ -177,26 +177,25 @@ This specification uses the terms "End-User", "Entity" as defined by
 OpenID Connect Core [OpenID.Core], the term "JSON Web Token (JWT)"
 defined by JSON Web Token (JWT) {{RFC7519}},
 the term "CBOR Web Token (CWT)" defined in {{RFC8392}}, "Client" as
-defined {{RFC6749}}, "Verifiable Presentation" defined in [OpenID4VP].
+defined {{RFC6749}}, "Holder", "Verifiable Presentation"
+defined in [OpenID4VP].
 
 Digital Credential:
-: A set of one or more claims about a subject made by an Issuer.
+: A set of one or more claims about a subject issued by a Credential Issuer.
```
Comment (peppelinux) [inline draft-demarco-oauth-status-assertions.md:184]:
```suggestion
: A set of one or more claims about a subject issued by an Issuer.
```

File: draft-demarco-oauth-status-assertions.md [hunk 12]
```diff
@@ -124,46 +124,46 @@ without requiring to query any third-party entities.
 Status Assertions show the status of digital
 credentials, whether in JSON Web Tokens (JWT) or CBOR Web Tokens (CWT)
 format. Status Assertions function
-similarly to OCSP Stapling ([RFC6066]), allowing clients to present to the
-relying parties
-time-stamped assertions provided by the issuer.
+similarly to OCSP Stapling ([RFC6066]), allowing Holders
+to present to the relying parties
```
Comment (peppelinux) [inline draft-demarco-oauth-status-assertions.md:128]:
```suggestion
to present to the Relying Parties
```

File: draft-demarco-oauth-status-assertions.md [hunk 13]
```diff
@@ -124,46 +124,46 @@ without requiring to query any third-party entities.
...
+time-stamped assertions provided by the Issuer.
 The approach outlined in this specification enables the
 verification of credentials against revocation without
```
Comment (peppelinux) [inline draft-demarco-oauth-status-assertions.md:131]:
```suggestion
verification of Credentials against revocation without
```

File: draft-demarco-oauth-status-assertions.md [hunk 14]
```diff
@@ -124,46 +124,46 @@ without requiring to query any third-party entities.
...
+|     Holder     |                              |      Issuer      |
 |                | Status Assertions            |                  |
 |                |<-----------------------------|                  |
 +----------------+                              +------------------+
 ~~~
 **Figure 1**: Status Assertion Issuance Flow.
 
-The figure below illustrates the process by which a client
+The figure below illustrates the process by which a Holder
 presents the Status Assertion along with the corresponding digital credential.
```
Comment (peppelinux) [inline draft-demarco-oauth-status-assertions.md:152]:
```suggestion
presents the Status Assertion along with the corresponding Digital Credential.
```

**Review by marinaado — APPROVED** [review]

**Pull request merged**